Musicplayer.css

```diff
@@ -67,62 +41,152 @@ li{
 }
 
 
+
 nav a{
 	display:block;
 	width:100px;
 	padding: 20px 15px;
 	text-decoration:none;
-	font-family:Arial;
-	color:black;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:bold;
+	color:white;
 	text-align: center;
 }
 
 nav a:hover{
-	background:purple;
+	background:black;
 	transition:0.5s;
 }
 
-.content_1{
-	background-color: white;
+ul li ul{
```

Comment:
21/9: I making the drop down menu which it was one on the techniques I decided to apply in my music database. So when the user hover in a link that contain so link then it will display some rows underneath the link that the user was hover on and this will help to minimize about the amount of link that my music database contain. So it will help to make the nav bar to be more clear and easy to follow. I create two drop down one is for contain all song but sort by a difference way and the second drop down are for the setting which can only uses by admin.